neurodiffeq/networks.py

```diff
@@ -24,10 +24,10 @@ class FCNN(nn.Module):
     """
 
     def __init__(self, n_input_units=1, n_output_units=1, n_hidden_units=None, n_hidden_layers=None,
-                 actv=nn.Tanh, hidden_units=None):
+                 actv=nn.Tanh, hidden_units=None, BN = False, last_actv = None):
```

Comment:
Documentation needed for `BN` and `last_actv`. Also let's use lowercase argument names.